src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.targets

```diff
@@ -208,6 +208,7 @@ The .NET Foundation licenses this file to you under the MIT license.
       <IlcArg Include="@(RdXmlFile->'--rdxml:%(FullPath)')" />
       <IlcArg Include="@(TrimmerRootDescriptor->'--descriptor:%(FullPath)')" />
       <IlcArg Condition="'$(NativeLib)' != ''" Include="--nativelib" />
+      <IlcArg Condition="'$(CustomNativeMain)' == 'true'" Include="--split-exe-initalization" />
```

Comment:
I don't have a good suggestion, just an observation: none of the other command line parameters have separate words with dashes.

Reply:
I shortened the flag to `--splitinit`, which is more in line with the other flags.